package.json

```diff
@@ -82,5 +82,10 @@
   },
   "engines": {
     "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
+  },
+  "config": {
+    "node": {
+      "version": "18.20.5"
```

Comment:
Maybe we should just pass the Node version as an explicit parameter in [the `build` npm script]()?

```diff
  "scripts": {
-   "build": "node build/build.mjs",
+   "build": "node build/build.mjs 18.20.5",
```

Reply:
Done :)